requirements.txt

```diff
@@ -16,6 +16,7 @@ platformdirs==4.3.6
 pre_commit==4.0.1
 pycodestyle==2.12.1
 pyflakes==3.2.0
+pytest==7.4.4
```

Comment:
I've been running all the tests using `PYTHONPATH=./src python -m unittest` from the base directory. Could you describe what the `pytest` version does differently? I'm somewhat hesitant to add a pretty significant testing library that were not really using except to run a command, but I do think there could be a case for making the jump from `unittest` to `pytest` depending on what we're getting out of it.

Reply:
I didn't realize the two do the same thing. I've just been running individual tests and was looking for a way to run them all. I removed pytest because I agree we have no use for it.